### PR TITLE
Feat quiz

### PR DESCRIPTION
**Pull request by abedfetrat**

### CODE REVIEW

File: CWback/appsettings.json
Comment (rorychatt) [inline CWback/appsettings.json:1]: haha ez password

File: CWfront/src/routeTree.gen.ts
Comment (rorychatt) [inline CWfront/src/routeTree.gen.ts:1]: you can gitignore this

Comment (abedfetrat) [reply CWfront/src/routeTree.gen.ts:1]: Good idea!

File: CWback/appsettings.json
Comment (abedfetrat) [reply CWback/appsettings.json:1]: Haha 😄 it's for my local DB

**Pull request merged**